cola-tests/src/main/java/com/github/bmsantos/core/cola/utils/ColaUtils.java

```diff
@@ -78,11 +78,11 @@ public static String binaryToResource(final String binaryFormat) {
     }
 
     public static String binaryToResourceClass(final String binaryFormat) {
-        String result = binaryToResource(binaryFormat);
-        if (!isClassFile(result)) {
-            result += CLASS_EXT;
+        StringBuilder result = new StringBuilder(binaryToResource(binaryFormat));
+        if (!isClassFile(result.toString())) {
+            result.append(CLASS_EXT);
```

Comment:
in fact, in these samples, I think that there will not exist any performance gain since only one "new StringBuilder()"will be created in the if statement.